graspologic/inference/latent_distribution_test.py

```diff
@@ -86,6 +88,17 @@ def latent_distribution_test(
     n_bootstraps : int (default=200)
         Number of bootstrap iterations for the backend hypothesis test.
         See :class:`hyppo.ksample.KSample` for more information.
+    
+    random_state : {None, int, `~np.random.RandomState`, `~np.random.Generator`}
+        This parameter defines the object to use for drawing random
+        variates.
+        If `random_state` is ``None`` the `~np.random.RandomState` singleton is
+        used.
+        If `random_state` is an int, a new ``RandomState`` instance is used,
+        seeded with `random_state`.
+        If `random_state` is already a ``RandomState`` or ``Generator``
+        instance, then that object is used.
+        Default is None.
 
     workers : int (default=1)
```

Comment:
```suggestion
    n_jobs : int (default=1)
```
For consistency.

Reply:
also add this to end of parameter description: "See :class:`joblib.Parallel` for more details."

Reply:
if using joblib the default should also be `None` and not 1.

Reply:
looks like you just need to update here https://github.com/microsoft/graspologic/blob/c816c420facc614ad7af777040b25d7553917788/graspologic/inference/latent_distribution_test.py#L280